helix-query.yaml

```
@@ -116,3 +116,22 @@ indices:
         select: head > meta[name="publication-date"]
         value: |
           attribute(el, 'content')
+
+    hrGlossary:
+    include:
+      - '/hr-glossary/**'
+    target: /hr-glossary/query-index
+    properties:
+      title:
+        select: head > meta[property="og:title"]
+        value: |
+          attribute(el, 'content')
+      robots:
+        select: head > meta[name="robots"]
+        value: |
+          attribute(el, 'content')
+      lastModified:
+        select: none
+        value: |
+          parseTimestamp(headers['last-modified'], 'ddd, DD MMM YYYY hh:mm:ss GMT')
+    
```

```suggestion
      include:
        - '/hr-glossary/**'
      target: /hr-glossary/query-index
      properties:
        title:
          select: head > meta[property="og:title"]
          value: |
            attribute(el, 'content')
        robots:
          select: head > meta[name="robots"]
          value: |
            attribute(el, 'content')
        lastModified:
          select: none
          value: |
            parseTimestamp(headers['last-modified'], 'ddd, DD MMM YYYY hh:mm:ss GMT')
      
```
Nicer with proper indentation.